package.json

```diff
@@ -27,7 +29,8 @@
   "scripts": {
     "test": "ember test",
     "start": "ember test --server",
-    "build": "ember build"
+    "build": "ember build",
+    "bench": "ember build && node ./bench/index.js"
```

Comment:
What do y'all think about a custom build to a `tmp` directory for benchmarking? If a user makes some changes to the lib to update perf, it's slightly more effort for them to undo the builds before committing their work. Building to a gitignored tmp directory, then running tests against that, adds a lil more complexity to the repo in exchange for user convenience.